modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/PaginationBarTag.java

```diff
@@ -40,6 +41,14 @@
 	public int doStartTag() throws JspException {
 		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
 
+		if (getPaginationBarDeltas() == null) {
+			setPaginationBarDeltas(
+				Arrays.asList(
+					new PaginationBarDelta(10), new PaginationBarDelta(20),
```

Comment:
Are this constants defined anywhere else?

Reply:
No, I just copied this from [here](https://github.com/liferay/clay/blob/d8251c47262ccff9e715baf050c2713b075b11fa/packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx#L14-L27) :see_no_evil:

Reply:
Being in JS only it's Ok, it was to see if we could extract them somewhere.

Reply:
Ok, that's what I had imagined you were asking for